### PR TITLE
Use ubuntu:16.04 as the base image

### PR DESCRIPTION
**Pull request by imikushin**
Use ubuntu:16.04 as the base image

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -14,9 +13,8 @@ RUN apt-get update && apt-get install -y \
     vim-tiny \
     openssl \
     libssl-dev \
-    software-properties-common && \
-    add-apt-repository ppa:vbernat/haproxy-1.6 && \
-    apt-get update && apt-get install -y haproxy && \
+    software-properties-common \
```
Comment (ibuildthecloud) [inline Dockerfile:18]: Missing `&&`.

File: Dockerfile [hunk 2]
```diff
@@ -14,9 +13,8 @@ RUN apt-get update && apt-get install -y \
...
+    haproxy && \
```
Comment (ibuildthecloud) [inline Dockerfile:19]: Can you confirm this is haproxy 1.6?

File: Dockerfile (hunk 1 above)
Comment (imikushin) [inline Dockerfile:18]: Nope, `haproxy` on the next line is continuing the packages list.

File: Dockerfile (hunk 2 above)
Comment (imikushin) [inline Dockerfile:19]: Yes, it is:

```
# haproxy -v
HA-Proxy version 1.6.3 2015/12/25
Copyright 2000-2015 Willy Tarreau <willy@haproxy.org>
```

**Comment by ruimarinho** [comment]
Is this ready for merge?

**Comment by cjellick** [comment]
Very stale. Closing

**Pull request closed without merging**